sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubWebJobsBuilderExtensions.cs

```diff
@@ -96,22 +96,22 @@ internal static void ConfigureInitialOffsetOptions(EventHubOptions options)
                         options.EventProcessorOptions.DefaultStartingPosition = EventPosition.Latest;
                         break;
                     case OffsetType.FromEnqueuedTime:
-                        try
+                        if (!options.InitialOffsetOptions.EnqueuedTimeUtc.HasValue)
                         {
-                            options.EventProcessorOptions.DefaultStartingPosition = EventPosition.FromEnqueuedTime(options.InitialOffsetOptions.EnqueuedTimeUtc.Value);
-                        }
-                        catch (FormatException fe)
-                        {
-                            string message = $"{nameof(EventHubOptions)}:{nameof(InitialOffsetOptions)}:{nameof(InitialOffsetOptions.EnqueuedTimeUtc)} is configured with an invalid format. " +
-                                "Please use a format supported by DateTime.Parse().  e.g. 'yyyy-MM-ddTHH:mm:ssZ'";
-                            throw new InvalidOperationException(message, fe);
+                            throw new InvalidOperationException(
```

Comment:
I like the `InvalidOperationException`, personally... just wanted to make sure that the change of exception type was intentional.

Reply:
What do you mean, it was always InvalidOperationException - there is just no longer an inner FormatException.

Reply:
You're right;  that's my bad.   I missed the catch/wrap.